api/controllers/articles.controller.js

```diff
@@ -14,4 +14,18 @@ exports.getArticlesById = (req, res, next) => {
 exports.getArticles = (req, res) => {
     fetchAllArticles().then((data) => res.status(200).send(data))
 }
-  
\ No newline at end of file
+
+exports.getArticlesComments = (req, res, next) => {
+  const articleId = req.params.article_id;
+  fetchArticleById(articleId)
+    .then((data) => {
+      if (data) return fetchArticlesComments(articleId);
+      else Promise.reject({status: 404, msg:'Article not found'});
+    })
+    .then((data) => {
+      res.status(200).send(data);
```

Comment:
Your data needs to be sent on a relevant key i.e.
`res.status(200).send({comments:data});`
This will mean your tests need updating to check the information given on this key
Can you also think of a more descriptive variable name than `data`?